packages/template/contentlayer.config.js

```diff
@@ -65,6 +69,22 @@ const Blog = defineDocumentType(() => ({
   computedFields,
 }));
 
+export const Author = defineDocumentType(() => ({
+  name: "Author",
+  filePathPattern: "authors/**/*.md*",
+  contentType: "mdx",
+  fields: {
+    name: {
+      type: "string",
+      required: true,
+    },
+    avatar: {
+      type: "string",
```

Comment:
This should have a default value. And while I like the one you picked, let's use that boring gray and white icon with the shape of the head 😅